src/js/gamestate.js

```diff
@@ -661,4 +675,12 @@ class GameState {
 
 }
 
-module.exports = GameState;
+// stop a card from attacking
```

Comment:
maybe Card needs to be an object, huh. this function sure feels ugly. a comment explaining why there are no parallelism bugs in this function would be nice maybe?